<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindowClass</class>
 <widget class="QMainWindow" name="MainWindowClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>651</width>
    <height>443</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>FON Studio</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>:/phone-icon.png</normaloff>:/phone-icon.png</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLineEdit" name="initFileName">
          <property name="text">
           <string>e:\dev\iland\fonstudio\inits\fon.xml</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="saveFile">
          <property name="text">
           <string>Save</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="applyXML">
          <property name="text">
           <string>Apply!</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QTextEdit" name="iniEdit">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maximumSize">
         <size>
          <width>1000</width>
          <height>16777215</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTabWidget" name="tabWidget">
        <property name="currentIndex">
         <number>0</number>
        </property>
        <widget class="QWidget" name="tab">
         <attribute name="title">
          <string>Stamping</string>
         </attribute>
         <widget class="QPushButton" name="stampTrees">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>10</y>
            <width>75</width>
            <height>23</height>
           </rect>
          </property>
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
          <property name="text">
           <string>stamp trees</string>
          </property>
         </widget>
         <widget class="QPushButton" name="pbRetrieve">
          <property name="geometry">
           <rect>
            <x>90</x>
            <y>10</y>
            <width>75</width>
            <height>23</height>
           </rect>
          </property>
          <property name="text">
           <string>retrieve</string>
          </property>
         </widget>
         <widget class="QPushButton" name="pbAddTrees">
          <property name="geometry">
           <rect>
            <x>190</x>
            <y>50</y>
            <width>75</width>
            <height>23</height>
           </rect>
          </property>
          <property name="text">
           <string>Add</string>
          </property>
         </widget>
         <widget class="QLineEdit" name="eTreeCount">
          <property name="geometry">
           <rect>
            <x>60</x>
            <y>50</y>
            <width>41</width>
            <height>20</height>
           </rect>
          </property>
          <property name="text">
           <string>100</string>
          </property>
         </widget>
         <widget class="QLineEdit" name="eTreeDBH">
          <property name="geometry">
           <rect>
            <x>140</x>
            <y>50</y>
            <width>41</width>
            <height>20</height>
           </rect>
          </property>
          <property name="text">
           <string>30</string>
          </property>
         </widget>
         <widget class="QLabel" name="label">
          <property name="geometry">
           <rect>
            <x>20</x>
            <y>50</y>
            <width>46</width>
            <height>14</height>
           </rect>
          </property>
          <property name="text">
           <string>Count</string>
          </property>
         </widget>
         <widget class="QLabel" name="label_2">
          <property name="geometry">
           <rect>
            <x>110</x>
            <y>50</y>
            <width>46</width>
            <height>14</height>
           </rect>
          </property>
          <property name="text">
           <string>DBH</string>
          </property>
         </widget>
        </widget>
        <widget class="QWidget" name="tab_2">
         <attribute name="title">
          <string>Testing</string>
         </attribute>
         <widget class="QWidget" name="layoutWidget">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>20</y>
            <width>255</width>
            <height>22</height>
           </rect>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_4">
           <item>
            <widget class="QLabel" name="label_3">
             <property name="text">
              <string>Formula</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="formula"/>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="layoutWidget_2">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>50</y>
            <width>255</width>
            <height>53</height>
           </rect>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <widget class="QLineEdit" name="lVar1">
             <property name="maximumSize">
              <size>
               <width>65</width>
               <height>16777215</height>
              </size>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="lVar2">
             <property name="maximumSize">
              <size>
               <width>65</width>
               <height>16777215</height>
              </size>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="lCalcResult">
             <property name="text">
              <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:8.25pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:8pt;&quot;&gt;Result&lt;/span&gt;&lt;a href=&quot;http://www.orf.at&quot;&gt;&lt;span style=&quot; text-decoration: underline; color:#0000ff;&quot;&gt;hoi&lt;/span&gt;&lt;/a&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="calcFormula">
             <property name="text">
              <string>Calculate</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="PaintArea" name="PaintWidget" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="cursor">
       <cursorShape>CrossCursor</cursorShape>
      </property>
      <property name="autoFillBackground">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>651</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>PaintArea</class>
   <extends>QWidget</extends>
   <header>paintarea.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
